repo2docker/buildpacks/conda/freeze.py

```diff
@@ -67,15 +69,15 @@ def freeze(env_file, frozen_file):
         f"continuumio/miniconda3:{MINICONDA_DOCKER_VERSION}",
         "sh", "-c",
         '; '.join([
-            'set -e',
+            'set -ex',
             f"conda install -yq conda={CONDA_VERSION}",
             'conda config --add channels conda-forge',
             'conda config --system --set auto_update_conda false',
-            f"conda env create -v -f /r2d/{env_file} -n r2d",
+            f"conda env create -v -f /r2d/{env_file_in_container} -n r2d",
```

Comment:
did something change that made these stop working? I feel like I've run the freeze script before and it worked OK...just wanna make sure that there isn't actually a different issue at play here

Reply:
I don't understand why this stopped working. But I also didn't go look at the history of the file to figure it out. The problem is that `env_file` now contains the full local path but we need the "relative to the path that got mounted" path.

Reply:
ecb4f28 addresses this. I'll mark this as resolved so we can start a new one if we need to.